packages/pluggableWidgets/datagrid-date-filter-web/src/DatagridDateFilter.editorPreview.tsx

```diff
@@ -0,0 +1,5 @@
+import { createElement, ReactElement } from "react";
+
+export function preview(): ReactElement {
+    return <div />;
```

Comment:
Dont we wanna do the same thing as textFilter ?

Reply:
Sorry I forgot to push the preview file. Fixed